ppa/archive/tests/test_gale.py

```diff
@@ -15,22 +16,32 @@
 @override_settings()
 def test_get_local_ocr(tmp_path):
     item_id = "CB0123456789"
-    page_num = "0001"
-    content = "Testing...\n1\n2\n3"
+    content = {"0001": "Testing...\n1\n2\n3"}
     # Mock ocr files for testing
-    ocr_dir = tmp_path.joinpath("147", item_id)
-    ocr_dir.mkdir(parents=True)
-    ocr_file = ocr_dir.joinpath(f"{item_id}_{page_num}0.txt")
-    ocr_file.write_text(content)
+    ocr_dir = tmp_path / "147"
```

Comment:
No need to change the code, but the `/` notation for `join_path` messes with me (because it immediately makes me worry about file systems)

Reply:
I tried to look to see if one of them was considered more pythonic or more idiomatic but didn't find much (didn't look very long). I like this syntax better because its terser.